push/views.py

```diff
@@ -9,13 +11,14 @@
 from push.services.send_webpush import SendWebPush
 from push.services.get_notification import GetNotification
 from push.services.save_subscription import SaveSubscription
+from push.tasks import task_send_web_push
 
 
 @api_view(['POST'])
 @permission_classes([AllowAny])
 def subscribe_client(request, *args, **kwargs):
     subscription_serializer = SubscriptionSerializer(data=request.data)
-    if subscription_serializer.is_valid():
+    if subscription_serializer.is_valid(raise_exception=True):
```

Comment:
```suggestion
    subscription_serializer.is_valid(raise_exception=True):
```
You don't need an `if`